assets/css/search.css

```diff
@@ -68,7 +68,7 @@ fieldset, legend {
 .filter-options label { margin: 0 10px; }
 
 
-#filters-apply { text-align: end; }
+#filters-apply { text-align: right; }
```

Comment:
Please keep `text-align: end;`, as it is required for Right-to-Left languages. I guess that you changed this because IE11 doesn't support `end`? if that's the case, you can do that:

```css
#filters-apply {
	text-align: right; /* IE11 only */
	text-align: end;   /* Override for compatible browsers */
}

Reply:
@SamantazFox FYI, Invidious in Hebrew and Arabic (most obvious RTL languages) don't change elements order :P

Reply:
not *yet* ^^ But it's something I need to fix!